scalyr_agent/config_util.py

```diff
@@ -85,16 +85,14 @@ def parse_array_of_strings(strlist, separators=[","]):
     return ArrayOfStrings(elems)
 
 
-NUMERIC_TYPES = set([int, long, float])
+NUMERIC_TYPES = set(six.integer_types + (float,))
```

Comment:
Remove extra comma.

Reply:
If there is only one element in tuple literal, without comma it will treat it as the element itself within redundant parentheses.

Reply:
For clarity:
>>> (2) + (2)
4
>>> (2,) + (2,)
(2, 2)